playbook/app/pb_kits/playbook/pb_caption/_caption.tsx

```diff
@@ -1,24 +1,22 @@
-/* @flow */
-
 import React from 'react'
 import classnames from 'classnames'
 import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
-import { deprecatedProps, globalProps } from '../utilities/globalProps'
+import { deprecatedProps, globalProps, GlobalProps } from '../utilities/globalProps'
```

Comment:
We have a `globalProps` and a `GlobalProps`? That feels a bit confusing

Reply:
It's a bit confusing for me as well. I know it saved me from having to specify a bunch of props 'GlobalProbs' already has.

Reply:
I'll get clarification and report back 🙃

Reply:
@RudeChowder nah `GlobalProps` is the type

Reply:
oppps.. I think I misunderstood what's actually going on here.. 👀 maybe we can talk about this one.  thanks @thestephenmarshall